oeps/oep-0004.rst

```diff
@@ -121,6 +121,21 @@ Governance
 * Resources should not, on the other hand, be too broad to be useful, i.e.,
   instead of ``courseware``, consider ``discussions`` or ``catalog`` or
   whatever specific resource is of interest
+ 
+* Scope descriptions need to be presented to the user by the token management
+  system.  They should therefore be maintained with the central list of scopes.
+  Scope descriptions should be internationalized, but should be presented with
+  the scope name, which is consistent across translations.
+
+Alt:
+
+* Scope metadata should be maintained centrally with the canonical list of 
+  scopes.
+
+* Scope descriptions should be internationalized strings, managed by Transifex.
+
+* Scopes should be presented to the user during authorization as a combination 
+  of the canonical name and the internationalized description.
```

Comment:
The first bulletpoint is more specifically focused on descriptions, rather than metadata, and provides more rationale.  Reading the rest of the OEP, I'm not sure that belongs here.  The alternate version is more prescriptive, and is generally applicable to metadata.  I'm leaning toward the alternate version, but interested in feedback.